src/components/PdfHighlighter.js

```diff
@@ -80,14 +80,7 @@ type Props<T_HT> = {
 
 const EMPTY_ID = "empty-id";
 
-const disableEvent = (event: Event) => {
-  event.preventDefault();
-  event.stopPropagation();
-  return false;
-};
-let clickTimeoutId: TimeoutID;
-
-class PdfHighlighter<T_HT: T_Highlight> extends Component<
+class PdfHighlighter<T_HT: T_Highlight> extends PureComponent<
```

Comment:
from: https://reactjs.org/docs/react-component.html#shouldcomponentupdate

> This method only exists as a performance optimization. Do not rely on it to “prevent” a rendering, as this can lead to bugs. Consider using the built-in PureComponent instead of writing shouldComponentUpdate() by hand. PureComponent performs a shallow comparison of props and state, and reduces the chance that you’ll skip a necessary update.

We need not to return false here in order to have `componentDidUpdate` called